<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditRewardDialog</class>
 <widget class="QDialog" name="EditRewardDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditReward</string>
  </property>
  <widget class="QCheckBox" name="enabledCheckBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Enabled</string>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="titleEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>45</number>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>520</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>520</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>DeleteReward</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>520</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="obsSourceComboBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="testObsSourceButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>200</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TestSource</string>
   </property>
  </widget>
  <widget class="QLabel" name="backgroundColor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>BackgroundColor</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="backgroundColorButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="limitRedemptionsPerStreamCheckBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>LimitRedemptionsPerStream</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="limitRedemptionsPerStreamSpinBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>360</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>99999999</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="limitRedemptionsPerUserPerStreamCheckBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>400</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>LimitRedemptionsPerUserPerStream</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="limitRedemptionsPerUserPerStreamSpinBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>400</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>99999999</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="globalCooldownEnabledCheckBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>430</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>EnableGlobalCooldown</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="globalCooldownSpinBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>440</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>604800</number>
   </property>
  </widget>
  <widget class="QLabel" name="uploadCustomIconLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>480</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>UploadCustomIconHere</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="timesLabel_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>400</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Times</string>
   </property>
  </widget>
  <widget class="QLabel" name="timesLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>360</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Times</string>
   </property>
  </widget>
  <widget class="QLabel" name="costLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cost</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="costSpinBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="cannotEditRewardLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>621</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red</string>
   </property>
   <property name="text">
    <string>CannotEditThisReward</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="updateObsSourcesButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="mediaSourceLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>51</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>51</height>
    </size>
   </property>
   <property name="text">
    <string>MediaSource</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="globalCooldownTimeUnitComboBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>440</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Seconds</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Minutes</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hours</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Days</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="descriptionLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="descriptionEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="randomPositionEnabledCheckBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>EnableRandomPosition</string>
   </property>
  </widget>
  <widget class="QLabel" name="secondsLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>320</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Seconds</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="loopVideoEnabledCheckBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>LoopVideoAndStopAfter</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="loopVideoDurationSpinBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>320</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>0.500000000000000</double>
   </property>
   <property name="maximum">
    <double>99.900000000000006</double>
   </property>
   <property name="value">
    <double>5.000000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
